src/locales/de.json

```
@@ -478,27 +478,51 @@
             }
         },
         "UpdatePanel": {
-            "CheckForUpdates": "auf aktualisierungen prüfen",
+            "Abort": "abbrechen",
+            "AreYouSure": "Bist du dir sicher?",
+            "CheckForUpdates": "auf Aktualisierungen prüfen",
+            "Close": "schließen",
+            "CommitHistory": "Commit Verlauf",
             "Commits": "Commits",
             "CommitsAvailable": "keine Einreichung verfügbar | {count} Einreichung verfügbar | {count} Einreichungen verfügbar",
-            "CommitsOnDate": "Engereicht am {date}",
+            "CommitsOnDate": "Eingereicht am {date}",
             "CommittedDaysAgo": "eingereicht vor {days} Tagen",
             "CommittedHoursAgo": "eingereicht vor {hours} Stunden",
             "CommittedOnDate": "eingereicht am {date}",
             "CommittedYesterday": "eingereicht gestern",
+            "ConfigChanges": "Konfigurationsänderungen",
+            "CountPackagesCanBeUpgraded": "{count} Pakete können aktualisiert werden",
             "Detached": "abgetrennt",
             "Dirty": "kompromittiert",
-            "ERROR": "FEHLER",
+            "GenericUpdateQuestion": "Überprüfe den Commit-Verlauf (falls verfügbar) und die GitHub-Projektseite von diesem Projekt, um festzustellen, ob manuelle Anpassungen nach dieser Aktualisierung erforderlich sind.",
+            "GitHubPage": "GitHub Projektseite",
+            "HardRecovery": "Harte Wiederherstellung",
+            "InitUpdateManager": "Der Update Manager ist noch nicht initialisiert. Dies ist normal, wenn das System zum ersten Mal gestartet wird. Bitte klicke auf die Aktualisierungsschaltfläche, um alle Komponenten zu initialisieren.",
             "Invalid": "ungültig",
+            "IUnderstandTheRisks": "Ich habe die Risiken verstanden",
+            "LinkToGithub": "Link zu GitHub",
+            "KlipperUpdateQuestionFirmware": "Dadurch wird die Klipper-Host-Software aktualisiert. Steuerplatinen, auf die du die Klipper-Firmware geflasht hast, müssen möglicherweise neu kompiliert und geflasht werden, bevor du deine Maschine wieder benutzen kannst.",
+            "KlipperUpdateQuestionConfig": "Dieses Update kann auch Änderungen an den Konfigurationsparametern enthalten, die in der Datei printer.cfg geändert werden müssen; Informationen hierzu findest du im Änderungsprotokoll.",
+            "MoonrakerUpdateQuestion": "Dadurch wird die Moonraker-API aktualisiert. Möglicherweise sind Änderungen an der Datei moonraker.conf erforderlich, damit du deinen Drucker wieder verwenden kannst.",
```

```suggestion
            "MoonrakerUpdateQuestion": "Dadurch wird die Moonraker-API aktualisiert. Möglicherweise sind Änderungen an der Datei moonraker.conf erforderlich, damit die Maschine wieder verwendet werden kann.",
```

Here we are talking about "Drucker". Further above we talk about "Machine". I think we should decide on one term, instead of using different ones in different locations but for the same "thing".